plugins/catalog/src/components/EntitySwitch/conditions.ts

```diff
@@ -14,57 +14,67 @@
  * limitations under the License.
  */
 
-import {
-  ComponentEntity,
-  ResourceEntity,
-  Entity,
-} from '@backstage/catalog-model';
+import { Entity } from '@backstage/catalog-model';
 
-function strCmp(a: string | undefined, b: string | undefined): boolean {
+export interface EntityPredicates {
```

Comment:
```suggestion
/** @public */
export interface EntityPredicates {
```

Reply:
Let's make sure this is marked `@public` so that we don't get a warning in the API report. Bonus points for adding a tiny bit of documentation 😄